www-client/qutebrowser/qutebrowser-1.7.0.recipe

```diff
@@ -27,7 +27,7 @@ REQUIRES="
 	pypeg2_python3
 	pyqt_python3
 	pyyaml_python3
-	setuptools_python3
+	setuptools_python36
```

Comment:
This is also needed for jinja_python3 vs jinja_python36 :)

Reply:
Resolved